etc/paella/mh_default_org/config.json

```diff
@@ -0,0 +1,148 @@
+{
+	"player":{
+		"accessControlClass": "OpencastAccessControl",
+		"profileFrameStrategy": "paella.ProfileFrameStrategy",
+		"videoQualityStrategy": "paella.BestFitVideoQualityStrategy",
+		"reloadOnFullscreen": true,
+		"videoZoom": {
+			"enabled":true,
+			"max":800,
+			"minWindowSize":500
+		},		
+        "methods": [
+			{ "factory":"ChromaVideoFactory", "enabled":true },
+			{ "factory":"WebmVideoFactory", "enabled":true },
+			{ "factory":"Html5VideoFactory", "enabled":true },
+			{ "factory":"MpegDashVideoFactory", "enabled":true },
+			{ "factory":"HLSVideoFactory", "enabled":true },
+			{ "factory":"RTMPVideoFactory", "enabled":true },
+			{ "factory":"ImageVideoFactory", "enabled":true },
+			{ "factory":"YoutubeVideoFactory", "enabled":true },
+			{ "factory":"Video360ThetaFactory", "enabled":true },
+			{ "factory":"Video360Factory", "enabled":true }
+        ],
+		"audioMethods":[
+			{ "factory":"MultiformatAudioFactory", "enabled":true }
+		],        
+        "audio": {
+            "master": 1.0,
+            "slave": 0.0
+        },
+        "rtmpSettings": {
+            "bufferTime": 5,
+            "requiresSubscription": false
+        },
+        "slidesMarks": {
+            "enabled": true,
+            "color": "gray"
+        }
+	},
+    "folders": {
+        "profiles": "config/profiles",
+        "resources": "resources",
+        "skins": "resources/style"
+    },
+	
+	"defaultProfile":"slide_professor",
+	"data":{
+		"enabled":true,
+		"dataDelegates":{
+			"default":"MHAnnotationServiceDefaultDataDelegate",
+			"trimming": "MHAnnotationServiceTrimmingDataDelegate",
+			"userInfo": "UserDataDelegate",
+			"metadata":"VideoManifestMetadataDataDelegate",
+			
+			"footprints": "MHFootPrintsDataDelegate",
+			"captions": "MHCaptionsDataDelegate",
```

Comment:
@miesgre, I didn't see a MHCaptionsDataDelegate in the 04_datadelegates.js.  Does it make sense to add one there? I can make a ticket/pull (eventually)

Something like this https://github.com/harvard-dce/paella-matterhorn/blob/master/paella-opencast/plugins/es.upv.paella.opencast.loader/04_datadelegates.js#L228

But it would also looks for 'captions/vtt'

'captions/timedtext' using  paella.matterhorn.DFXPParser();

'captions/vtt' to use paella.captions.parsers.WebVTTParserPlugin();

Reply:
@miesgre Is the https://github.com/polimediaupv/paella-opencast repository going to go dormant now, and all related comments for  paella-opencast bridging should be made to Opencast Jira and this repo?

Reply:
@karendolan Yes, the idea is to disable the https://github.com/polimediaupv/paella-opencast repository. All issues should be in Jira and all development to paella-opencast bindings should be done here.

About captions. At the moment paella is not showing captions in opencast. The reason is that I don't have a mp with captions and I don't know how an attachment is added to the mp. Is it a catalog? an attachment?. Is there a mp example I can upload to my dev machine to test?

Reply:
@miesgre,  We've been using Paella for Opencast captions for a long time. We adopted Opencast's original way of considering caption files as a "catalog". They did that probably because captions are/were traditionally in XML format, like mpeg-7 and dublincore.

Here is an example MP that we use for testing in both 1.6x and 5x. https://github.com/harvard-dce/paella-matterhorn/blob/master/fixtures/example-captions-episode.json#L287

Here is the example caption file that we've been using (Timed Text Authoring Format V1.0 DXFP):  https://github.com/harvard-dce/paella-matterhorn/blob/master/fixtures/captions.dfxp

This is how we currently parsing (the hard-coded) flavor "captions/timedtext"  for Paella consumption: https://github.com/harvard-dce/paella-matterhorn/blob/master/paella-opencast/plugins/es.upv.paella.opencast.loader/04_datadelegates.js#L228

Reply:
@miesgre, It would be convenient if the flavor to parser designation can be pulled from the config. 

For example: 
"captions/timedtext": "paella.matterhorn.DFXPParser",
"captions/vtt": "paella.captions.parsers.WebVTTParserPlugin"

Reply:
Our site anticipates uploading caption catalogs of different types, so there may be either a "captions/timedtext" or a "captions/vtt", or both, or some future flavor.